examples/draw_and_compare_graphs_with_same_layout.py

```diff
@@ -0,0 +1,51 @@
+"""
+.. _ex-draw-graphs:
+
+=============================================================
+Drawing graphs and setting their layout for visual comparison
+=============================================================
+
+One can draw a graph without setting the `pos` argument, in that case graphviz will choose how to place the nodes.
+See https://graphviz.readthedocs.io/en/stable/api.html?highlight=render#graphviz.Digraph.render
+
+If one sets the `pos` argument, the positions will be fixed.
```

Comment:
```suggestion
If one sets the ``pos`` argument, the positions will be fixed.
```

Single '`' ticks make sphinx look for a reference.

Reply:
Actually, I would suggest we do not even need this portion. The docstring of the function should be self-documenting.